zerver/openapi/zulip.yaml

```diff
@@ -1225,657 +1263,3511 @@ paths:
+                                  properties:
+                                    user_id:
+                                      type: integer
+                                      description: |
+                                        The id of the user.
+                                    email:
+                                      type: string
+                                      description: |
+                                        The email of the user.
+                                recipients:
+                                  type: array
+                                  description: |
+                                    Array of dictionaries where each dictionary contains the
+                                    email and user_id of a single user who will receive the
+                                    typed message.
+                                  items:
+                                    type: object
+                                    additionalProperties: false
+                                    description: |
+                                      Object containing user id and email of a recipient.
+                                    properties:
+                                      user_id:
+                                        type: integer
+                                        description: |
+                                          The id of the user.
+                                      email:
+                                        type: string
+                                        description: |
+                                          The email of the user.
+                              example:
+                                {
+                                  "type": "typing",
+                                  "op": "start",
+                                  "sender":
+                                    {
+                                      "user_id": 10,
+                                      "email": "user10@zulip.testserver",
+                                    },
+                                  "recipients":
+                                    [
+                                      {
+                                        "user_id": 8,
+                                        "email": "user8@zulip.testserver",
+                                      },
+                                      {
+                                        "user_id": 10,
+                                        "email": "user10@zulip.testserver",
+                                      },
+                                    ],
+                                  "id": 0,
+                                }
+                            - type: object
+                              additionalProperties: false
+                              description: |
+                                Event sent to a user when flags[message flags][message-flags] are added
```

Comment:
Does this generate links that work?

Reply:
This flag description has been copied from the flag description in get /messages so I think it should work.

Reply:
Hmm, I'm skeptical of the syntax.